expirationd.lua

```diff
@@ -603,6 +608,7 @@ local function expirationd_run_task(name, space_id, is_tuple_expired, options)
         args = '?',
         atomic_iteration = '?boolean',
         force = '?boolean',
+        force_broken = '?boolean',
```

Comment:
I think that 'functional index' words should be in the option name. Say, `forcely_allow_functional_index`. A long name is okay for a dangerous option of the I-know-what-I-an-doing kind, IMHO.

Reply:
No problems, I changed it to ```force_allow_functional_index```